dist-persist/wbstack/src/Settings/LocalSettings.php

```diff
@@ -461,6 +462,7 @@ function onSkinTemplateNavigationUniversal( SkinTemplate $skin, array &$links )
 $wgReCaptchaSendRemoteIP = true;
 $wgReCaptchaSiteKey = getenv('MW_RECAPTCHA_SITEKEY');
 $wgReCaptchaSecretKey = getenv('MW_RECAPTCHA_SECRETKEY');
+$ceAllowConfirmedEmail = true;
```

Comment:
Why is this still needed? If I get this https://gerrit.wikimedia.org/r/c/mediawiki/extensions/ConfirmEdit/+/348224/ right, `$wgAllowConfirmedEmail` should work on its own? Or are we on a very old version?

Reply:
Seems we're on a version that should be content with the `$wg` version https://github.com/wbstack/mediawiki/blob/8d0ba2f27d2c4bfedf0e5900f59c2e443f08f6d1/dist/extensions/ConfirmEdit/extension.json#L161-L163